tests/test_staggered_utils.py

```diff
@@ -59,3 +76,87 @@ def test_is_param(ndim):
         for dd in d:
             avg = .5 * (avg + avg.subs({dd: dd - dd.spacing}))
         assert f2._eval_at(var).evaluate == avg
+
+
+@pytest.mark.parametrize('expr, expected', [
```

Comment:
Worth having a more intricate composite expression?

Reply:
as I wrote above, yes, and perhaps x-fail it if it needs to

Reply:
No